### ISSUE TITLE
2.0 Alpha 25: TypeError: Map constructor does not accept arguments in Safari 8 on Mac OS X and iPad (iOS 8.3)

### ISSUE THREAD

**Issue by jomeier** [angular/angular#2163]
```javascript
[Error] TypeError: Map constructor does not accept arguments

Map@[native code]
clone@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:7452:27
AppView@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:17776:58
createView@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:19920:44
createRootHostView@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:22742:48
https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:13666:67
run@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:115:22
https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:12418:43
zoneBoundFn@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:88:22
lib$es6$promise$$internal$$tryCatch@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:479:24
lib$es6$promise$$internal$$invokeCallback@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:491:52
lib$es6$promise$$internal$$publish@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:462:52
https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:195:17
microtask@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:12438:25
run@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:115:22
https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:12418:43
zoneBoundFn@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:88:22
flush@https://code.angularjs.org/2.0.0-alpha.25/angular2.dev.js:1208:13
	logError (angular2.dev.js, line 17335)
	defaultErrorReporter (angular2.dev.js, line 24768)
	_onError (angular2.dev.js, line 12456)
	onError (angular2.dev.js, line 12404)
	run (angular2.dev.js, line 118)
	(anonyme Funktion) (angular2.dev.js, line 12418)
	zoneBoundFn (angular2.dev.js, line 88)
	lib$es6$promise$$internal$$tryCatch (angular2.dev.js, line 479)
	lib$es6$promise$$internal$$invokeCallback (angular2.dev.js, line 491)
	lib$es6$promise$$internal$$publish (angular2.dev.js, line 462)
	(anonyme Funktion) (angular2.dev.js, line 195)
	microtask (angular2.dev.js, line 12438)
	run (angular2.dev.js, line 115)
	(anonyme Funktion) (angular2.dev.js, line 12418)
	zoneBoundFn (angular2.dev.js, line 88)
	flush (angular2.dev.js, line 1208)
```

In anular2.dev.js : 17335

Safari 8.0.6, Traceur 0.0.87, Mac OS X Yosemite (10.10.3) and also with iPad (iOS 8.3).

Page refuses to load. Nothing is displayed :-/

**Comment by jomeier** [comment]
Seems to be a bug in traceur: https://github.com/google/traceur-compiler/issues/1650 with Safari browsers?

Do we have to use traceur or is there another possibility for using angular 2 alpha?

**Comment by mlaval** [comment]
Fixed by #2096
